arkane/reference.py

```diff
@@ -369,6 +381,32 @@ def thermo_data(self, value):
             else:
                 raise ValueError('thermo_data for a CalculatedDataEntry object must be an rmgpy ThermoData object')
 
+    @property
+    def electronic_energy(self):
+        return self._electronic_energy
+
+    @electronic_energy.setter
+    def electronic_energy(self, value):
+        if value:
```

Comment:
Still not sure why `if value` is needed

Reply:
This is one of the ones where it could be None or an empty string if read from the supporting information file